core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/plugins/ai-content-prompt.plugin.ts

```diff
@@ -127,15 +127,13 @@ export class AIContentPromptView {
          * template in ai-content-prompt.component.html
          */
 
-        this.componentStore.status$
-            .pipe(
-                skip(1),
-                takeUntil(this.destroy$),
-                filter((status) => status === 'exit')
-            )
-            .subscribe(() => {
+        this.componentStore.status$.pipe(skip(1), takeUntil(this.destroy$)).subscribe((status) => {
+            if (status === 'exit') {
```

Comment:
What do you think about instead use a string, use the const enum about componentStates?

Reply:
agree ! done